### ISSUE TITLE
test_usart_c::receive always returns 0xAA

### ISSUE THREAD

**Issue by Lennart99** [R2D2-2019/USART_library#10]
The receive function of the test_usart class always returns 0xAA.
I think this limits the use cases of this class, because it isn't really possible to test the processing of usart messages of a module. I think it would be more useful to be able to prepare a message in the bus and let the module receive that message thru the usart library, a bit like the mock-bus of the internal communication library.

It would than look a bit like this in the test code, this is of course just an example:
```C++
test_usart_c usart;  // initialize test usart
usart.prepare_message({0xAA, 0xFF, 0x67, some_more_bytes}); // add a message to the usart to be processed by the module

// run code to be tested
module_to_test_c module(usart);
module.proces_uart();
// check stuff
```

for sending you could do something simular of course.

**Comment by PatrickDekker98** [comment]
Yes, this is correct, in a later version this will be added

**Comment by itzandroidtab** [comment]
This is implemented in the current PR #15
